leases/utils.py

```diff
@@ -63,3 +63,30 @@ def calculate_berth_lease_end_date() -> date:
     Leases always end on 14.9, the year depends on the current year
     """
     return calculate_season_end_date()
+
+
+def calculate_winter_storage_lease_start_date() -> date:
+    """
+    Return the date when the "winter" lease season should start
+
+    The season should start by default on 15.9 (one day after the "summer" season ends)
+    """
+
+    # Otherwise, return the latest date between the default start date or today
+    today = date.today()
+    default = max(date(day=15, month=9, year=today.year), today)
```

Comment:
this is good logic for the period from 10.6 till 15.9, i.e. during the summer season. however from 1.1 till 10.6 it should still use today, I think. otherwise, you might end up with a ws lease that start e.g. 15.9.2020, but ends 10.6.2020.

Reply:
So in this case it would be
```python
if today < date(10, 6, today.year):
    return min(date(day=15, month=9, year=today.year), today)
```